collection-scripts/gather_services_status

```diff
@@ -162,6 +165,25 @@ get_ceilometer_status() {
     fi
 }
 
+# Octavia service gathering - loadbalancers,
+get_octavia_status() {
+    local OCTAVIA_PATH="$BASE_COLLECTION_PATH/ctlplane/octavia"
+    mkdir -p "$OCTAVIA_PATH"
+    resources="amphora availabilityzone availabilityzoneprofile flavor flavorprofile healthmonitor l7policy "
+    resources="$resources listener pool provider quota"
+
+    for r in $resources; do
+        run_bg ${BASH_ALIASES[os]} loadbalancer $r list '>' "$OCTAVIA_PATH"/"${r}_list"
+    done;
+    # TODO(beagles): dump details of each item. This involves piping ids from a list commands into
```

Comment:
I see a couple of `TODO(s)` there: is the patch ready or do you need extra work?

Reply:
Patch is ready for now